packages/grafana-ui/src/components/Layout/Grid/Grid.internal.story.tsx

```diff
@@ -0,0 +1,356 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useTheme2 } from '../../../themes';
+import { Stack } from '../../../unstable';
+import { withCenteredStory } from '../../../utils/storybook/withCenteredStory';
+
+import { Grid } from './Grid';
+import mdx from './Grid.mdx';
+import { GridItem } from './GridItem';
+
+const gridItem = (index: number, theme: GrafanaTheme2, width?: string, height?: string) => {
+  const colorOrder = index >= 6 ? index - 6 : index;
+  const itemOrder = index + 1;
+  const colorList = [
+    theme.visualization.getColorByName('red'),
+    theme.visualization.getColorByName('orange'),
+    theme.visualization.getColorByName('purple'),
+    theme.visualization.getColorByName('blue'),
+    theme.visualization.getColorByName('yellow'),
+    theme.visualization.getColorByName('green'),
+  ];
```

Comment:
```suggestion
const colorList = ['red', 'orange', 'purple', 'blue', 'yellow', 'green'].map(theme.visualization.getColorByName);
```
Nit: This can be simplified a bit.